src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -31,12 +31,12 @@
    :witan/version "1.0"
    :witan/input-schema {:population PopulationSchema}
    :witan/param-schema {:_ nil}
-   :witan/output-schema {:latest-yr-popn PopulationSchema}}
+   :witan/output-schema {:latest-yr-popn PopulationSchema :loop-year s/Int}}
   [{:keys [population]} _]
   (let [latest-yr (get-last-yr-from-popn population)
         last-yr-data (i/query-dataset population {:year latest-yr})
```

Comment:
This also doesn't need to be queried for, as it is the data you created on the last iteration. Can just be stashed in the looping object :basis-for-next-year or something. Save a scan per loop!